### ISSUE TITLE
Flatpak support

### ISSUE THREAD

**Issue by RobLoach** [osztenkurden/steam-game-path#12]
Discover steam path for Flatpak

**Comment by konsumer** [comment]
For my system (Pop!OS 22.04) flatpak is integrated into package-manager, so it's a bit hard to tell where things come from (traditional ubuntu/debian apt or flatpak.) I think the directory it should look for is here:

```
~/.var/app/com.valvesoftware.Steam/.local/share/Steam/steamapps/
```

**Comment by osztenkurden** [comment]
Fixed in 2.3.0
